src/pages/workspace/accounting/intacct/export/SageIntacctNonReimbursableExpensesPage.tsx

```diff
@@ -0,0 +1,156 @@
+                : !!config?.export?.errorFields?.nonReimbursableCreditCardChargeDefaultVendor,
+        pendingAction:
+            config?.export.nonReimbursable === CONST.SAGE_INTACCT_NON_REIMBURSABLE_EXPENSE_TYPE.VENDOR_BILL
+                ? config?.export?.pendingFields?.nonReimbursableVendor
+                : config?.export?.pendingFields?.nonReimbursableCreditCardChargeDefaultVendor,
+    };
+
+    const defaultVendor = (
+        <OfflineWithFeedback
+            key={defaultVendorSection.description}
+            pendingAction={defaultVendorSection.pendingAction}
+        >
+            <MenuItemWithTopDescription
+                title={defaultVendorSection.title}
+                description={defaultVendorSection.description}
+                shouldShowRightIcon
+                onPress={defaultVendorSection.action}
+                brickRoadIndicator={defaultVendorSection.hasError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+            />
+        </OfflineWithFeedback>
+    );
+
+    const creditCardAccountSection = {
+        description: translate('workspace.sageIntacct.creditCardAccount'),
+        action: () => Navigation.navigate(ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_NON_REIMBURSABLE_CREDIT_CARD_ACCOUNT.getRoute(policyID)),
+        title: config?.export.nonReimbursableAccount ? config.export.nonReimbursableAccount : translate('workspace.sageIntacct.notConfigured'),
+        hasError: !!config?.export?.errorFields?.nonReimbursableAccount,
+        pendingAction: config?.export?.pendingFields?.nonReimbursableAccount,
+    };
+
+    const creditCardAccount = (
+        <OfflineWithFeedback
+            key={creditCardAccountSection.description}
+            pendingAction={creditCardAccountSection.pendingAction}
+        >
+            <MenuItemWithTopDescription
+                title={creditCardAccountSection.title}
+                description={creditCardAccountSection.description}
+                shouldShowRightIcon
+                onPress={creditCardAccountSection.action}
+                brickRoadIndicator={creditCardAccountSection.hasError ? CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR : undefined}
+            />
+        </OfflineWithFeedback>
+    );
+
+    return (
+        <ConnectionLayout
+            headerTitle="workspace.sageIntacct.nonReimbursableExpenses.label"
+            title="workspace.sageIntacct.nonReimbursableExpenses.description"
+            titleStyle={[styles.ph5, styles.pb5]}
+            onBackButtonPressRoute={ROUTES.POLICY_ACCOUNTING_SAGE_INTACCT_EXPORT.getRoute(policyID)}
+            accessVariants={[CONST.POLICY.ACCESS_VARIANTS.ADMIN, CONST.POLICY.ACCESS_VARIANTS.PAID]}
+            featureName={CONST.POLICY.MORE_FEATURES.ARE_CONNECTIONS_ENABLED}
+            displayName={SageIntacctNonReimbursableExpensesPage.displayName}
+            policyID={policyID}
+            connectionName={CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT}
```

Comment:
Add `shouldUseScrollView={false}` or switch to use `SelectionScreen` to avoid warning

```suggestion
            connectionName={CONST.POLICY.CONNECTIONS.NAME.SAGE_INTACCT}
            shouldUseScrollView={false}
```